generate/seccomp/seccomp_default_unsupported.go

```diff
@@ -0,0 +1,14 @@
+// +build !linux
+
+package seccomp
+
+// These are copied from linux/amd64 syscall values, as a reference for other
+// platforms to have access to
+const (
+	CLONE_NEWIPC  = 0x8000000
+	CLONE_NEWNET  = 0x40000000
+	CLONE_NEWNS   = 0x20000
+	CLONE_NEWPID  = 0x20000000
+	CLONE_NEWUSER = 0x10000000
+	CLONE_NEWUTS  = 0x4000000
```

Comment:
You may want to add [`CloneNewCgroup = 0x02000000`][1].  We support the cgroup namespace since opencontainers/runtime-spec#397.  I'm not sure why the seccomp lib doesn't need it (yet?), but I'd rather try to keep full new-namespace coverage.

[1]: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git/tree/include/uapi/linux/sched.h?h=v4.14.15#n25

Reply:
it's not even in the go syscall yet. I'm going to hold on this addition.

Reply:
> it's not even in the go syscall yet. I'm going to hold on this addition.

`syscall` is end-of-life.  It's [in golang.org/x/sys/unix][1] since golang/sys@b44883b47 (2016-06-09).

[1]: https://godoc.org/golang.org/x/sys/unix